homeassistant/components/vodafone_station/coordinator.py

```diff
@@ -0,0 +1,170 @@
+"""Support for Vodafone Station."""
+import asyncio
+from datetime import datetime, timedelta
+from typing import Any, TypedDict
+
+import aiohttp
+from aiovodafone.api import VodafoneStationApi, VodafoneStationDevice
+
+from homeassistant.components.device_tracker import DEFAULT_CONSIDER_HOME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+from homeassistant.util import dt as dt_util
+
+from .const import _LOGGER, DOMAIN
+
+
+class VodafoneStationDeviceInfo:
```

Comment:
This looks like it could be a dataclass

Reply:
Looks like this comment was missed

Reply:
I waited just because there are 2 functions in the class that I'm not sure how to better handle.

Reply:
example here https://github.com/bdraco/home-assistant/commit/f59acace06049649580668f4fc4d580f452e960e#diff-d0cc7431150692e4477b274ebbab67841665d4a3cb3df7e9563a1cbf8eae7339R20

Reply:
Done, thx !